src/index.jsx

```diff
@@ -121,7 +121,7 @@ initialize({
         ENABLE_UNIT_PAGE: process.env.ENABLE_UNIT_PAGE || 'false',
         ENABLE_ASSETS_PAGE: process.env.ENABLE_ASSETS_PAGE || 'false',
         ENABLE_VIDEO_UPLOAD_PAGE_LINK_IN_CONTENT_DROPDOWN: process.env.ENABLE_VIDEO_UPLOAD_PAGE_LINK_IN_CONTENT_DROPDOWN || 'false',
-        ENABLE_TAGGING_TAXONOMY_PAGES: process.env.ENABLE_TAGGING_TAXONOMY_PAGES || 'false',
+        DISABLE_TAGGING_FEATURE: process.env.DISABLE_TAGGING_FEATURE || 'false',
```

Comment:
Hi @yusuf-musleh!

I think getting the data from the backend is the right approach, but we need to fetch a lot of data to use this selector.

I created a new hook (using react-query) that solves that problem. The only issue is that we need this info in the index.jsx, where we don't have the necessary configs just to call a hook.

We may need to use some kind of "lazy load" for the routes. Maybe @bradenmacdonald could help us with some ideas?

Reply:
hey guys, I don't know if there is a good pattern for this now. I think we should just stick to the simplest option and have an environment variable that disables the MFE features (`getConfig().DISABLE_TAGGING_FEATURE`) and a waffle flag that controls the legacy UI (`content_tagging.disabled`).

The waffle flag is already set on edx.org, in anticipation of this PR, but we'll also have to get them to set the env var once it's confirmed.

Reply:
Actually I like @rpenido's approach and I think it's even better if we can get it to work. Which `index.jsx` is the problem? The studio home page or the taxonomies page?

Reply:
The [root index](https://github.com/openedx/frontend-app-course-authoring/blob/de408b5a3a9d66d8061f801e5ebe05ec7f277c08/src/index.jsx#L60), where we declare the `<App>`.

One "easy" approach to avoid this problem (needing a provider at the App level) is to move the check to the destination pages, showing an error message or redirecting to another place if the flag is disabled. I think this is not the best solution, but it will work.

Reply:
> One "easy" approach to avoid this problem (needing a provider at the App level) is to move the check to the destination pages, showing an error message or redirecting to another place if the flag is disabled. 

So let's go with that.

Reply:
OK, I implemented that ^

@yusuf-musleh could you please review/test this PR again?